src/spellbot/__init__.py

```diff
@@ -865,9 +865,10 @@ async def help(
             "💜 You can help keep SpellBot running by supporting me on Ko-fi! "
             "<https://ko-fi.com/Y8Y51VTHZ>"
         )
-        await message.channel.send(
-            s("dm_sent", reply=f"<@{cast(discord.User, message.author).id}>")
-        )
+        if not isinstance(message.channel, discord.channel.DMChannel):
```

Comment:
@crookedneighbor you can actually do like this and I think it will work with the mocks:

```suggestion
        if message.channel.type != "private":
```

Reply:
I did see that as an alternate. I thought there might be a conflict here with group dms, but now that I think about it, I don't think bots can be part of group dms.

(quite possible I'm just getting confused with the slack api)

Will try this out later tonight.